src/chart/inputlisteners/canvas-input-listener.component.ts

```diff
@@ -232,7 +232,7 @@ export class CanvasInputListenerComponent extends ChartBaseElement {
 				let timeoutId: number | null = null;
 				return () => {
 					if (timeoutId) {
-						this.dbClickSubject.next(this.currentPoint);
```

Comment:
It's intersting that Anastasia tried to subscribe to `dbClickSubject` before and it didn't work